### DISCUSSION TITLE
I think the commands prefix 'make: ' should be replaced by 'filament: '

### DISCUSSION THREAD

**Discussion by nastoychev** [filamentphp/filament#5559] [Ideas]
When I install filament the make commands are extended by the package, which I don't think should be the case. I would leave artisan make: only for laravel/framework.

Changing the name of the commands from make to another prefix. ( ex. filament:... ) 
It's quite confusing what comes after installing the package and as a user I want to see in a nice ordered overview (ex. when I type php artisan filament: ).

**Comment by danharrin** [comment]
This already exists, try `filament:resource` or `filament:page`. They are aliases
